src/main/scala/temple/generate/server/go/service/GoServiceMainGenerator.scala

```diff
@@ -5,30 +5,38 @@ import temple.generate.server.ServiceRoot
 import temple.generate.utils.CodeTerm
 import temple.generate.utils.CodeTerm.{CodeWrap, mkCode}
 import temple.utils.StringUtils.doubleQuote
+import temple.generate.server.go.common.GoCommonGenerator._
 
 import scala.Option.when
+import scala.collection.immutable.ListMap
 
 object GoServiceMainGenerator {
 
-  /** Given a service name, module name and whether the service uses inter-service communication, return the import
-    * block */
-  private[service] def generateImports(root: ServiceRoot, usesComms: Boolean): String = {
-    val standardImports = Seq(
-      //"flag",
-      //"log",
-      "net/http",
-    ).map(doubleQuote)
-
-    val customImports = Seq[CodeTerm](
-      when(usesComms) { s"${root.name}Comm ${doubleQuote(s"${root.module}/comm")}" },
-      s"${root.name}DAO ${doubleQuote(s"${root.module}/dao")}",
-      //doubleQuote(s"$module/util"),
-      //s"valid ${doubleQuote("github.com/asaskevich/govalidator")}",
-      //doubleQuote("github.com/gorilla/mux"),
+  private[service] def generateImports(root: ServiceRoot, usesTime: Boolean, usesComms: Boolean): String =
+    mkCode(
+      "import",
+      CodeWrap.parens.tabbed(
+        //doubleQuote("encoding/json"),
+        //doubleQuote("flag"),
+        //doubleQuote("fmt"),
+        //doubleQuote("log"),
+        //doubleQuote("net/http"),
+        //when(usesTime) { doubleQuote("time") },
+        //"",
```

Comment:
Why?

Reply:
> Omits some imports so integration tests pass, will be uncommented at a later date

Reply:
Integration tests fail on unused imports, and since the rest of the file isn't generated